pkg/instancetype/instancetype_test.go

```diff
@@ -304,7 +304,7 @@ var _ = Describe("Instancetype and Preferences", func() {
 				vm.Spec.Template.Spec.Domain.CPU = &v1.CPU{
 					Cores: 1,
 				}
-				Expect(instancetypeMethods.StoreControllerRevisions(vm)).To(MatchError(Equal(fmt.Sprintf(instancetypeErrors.VMFieldsConflictsErrorFmt, "spec.template.spec.domain.cpu.cores"))))
+				Expect(instancetypeMethods.StoreControllerRevisions(vm)).To(MatchError(conflict.New("spec", "template", "spec", "domain", "cpu", "cores").Error()))
```

Comment:
> MatchError succeeds if actual is a non-nil error that matches the passed in string, error, function, or matcher. 

Might be OK to drop `.Error()` from calls to `MatchError`.

https://pkg.go.dev/github.com/onsi/gomega#MatchError